core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageFeatureStep.java

```diff
@@ -127,11 +127,11 @@ void addExportsToNativeImage(BuildProducer<JPMSExportBuildItem> features,
             List<NativeImageResourcePatternsBuildItem> resourcePatterns) {
         // required in order to access org.graalvm.nativeimage.impl.RuntimeSerializationSupport and org.graalvm.nativeimage.impl.ConfigurationCondition
         features.produce(
-                new JPMSExportBuildItem("org.graalvm.sdk", "org.graalvm.nativeimage.impl", GraalVM.Version.VERSION_22_1_0));
+                new JPMSExportBuildItem("org.graalvm.sdk", "org.graalvm.nativeimage.impl", GraalVM.Version.CURRENT));
         // required in order to access com.oracle.svm.core.jni.JNIRuntimeAccess
         if (jniRuntimeAccessibleClasses != null && !jniRuntimeAccessibleClasses.isEmpty()) {
             features.produce(new JPMSExportBuildItem("org.graalvm.nativeimage.builder", "com.oracle.svm.core.jni",
-                    GraalVM.Version.VERSION_22_1_0));
+                    GraalVM.Version.CURRENT));
```

Comment:
This should not change. It says that the export should be done for GraalVM >= 22.1